cmake/projects/libsecp256k1/hunter.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020, Soramitsu Co., Ltd.
+# All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    libsecp256k1
+    VERSION
+    0.1
```

Comment:
Version mismatch with default.cmake.
It should be the same in both places - either 0.0.1 or 0.1

Reply:
It looks like 0.1 is the correct option